Integrate Sanity with Coaching

SawanKumarBhagat opened the pull request:
I have added the basic necessary setup for sanity with which we can store and get the notes from CMS in out website. Studio URL - https://sanjeevedupoint.sanity.studio/

SawanKumarBhagat committed:
Integrated Sanity

SawanKumarBhagat committed:
Adding @sanity/client in package

SawanKumarBhagat committed:
@sanity/client fix

SawanKumarBhagat committed:
Removed backend

SawanKumarBhagat committed:
Notes component basic UI setup

SawanKumarBhagat commented:
Hi @maliknajeeb113 , Please review the basic UI setup for notes page. Once done, setup the environment variables in the vercel for sanity keys and make the changes accordingly.

SawanKumarBhagat committed:
env setup

SawanKumarBhagat committed:
Added notes cards

SawanKumarBhagat committed:
added body page details

maliknajeeb113 reviewed (commented):
- checking on this - making some cosmetic changes 
- rest all works good
- will merge today

maliknajeeb113 committed:
fix: color for cards
fix: weird bug - infinte scroll cause the nav to not work
fix: padding for cards so they dont overlap

remove: padding from all sections
remove: outlet from router as not required

add: react-hamburger menu with animation in navbar

change: client.js to utils

maliknajeeb113 committed:
continuation of previous commit

maliknajeeb113 committed:
fix: padding and margins for the site
